jep/1/README.adoc

```diff
@@ -0,0 +1,542 @@
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+= JEP-1: Jenkins Enhancement Proposal Format
+
+:toc:
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 1
+
+| Title
+| Jenkins Enhancement Proposal Format
+
+| Author
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| Status
+| :speech_balloon: Draft
+
+| Type
+| Process
+
+| Created
+| 2017-09-12
+
+| Discussion-To
+| link:https://groups.google.com/d/msg/jenkinsci-dev/spDAr8EJm3c/T9Nmhn-fAQAJ[jenkinsci-dev@]
+
+|===
+
+
+== What is a JEP?
+
+JEP stands for Jenkins Enhancement Proposal. A JEP is a design
+document providing information to the Jenkins community, or describing
+a new feature for Jenkins or its processes or environment. The JEP
+should provide a concise technical specification of the feature and a
+rationale for the feature.
+
+We intend JEPs to be the primary mechanisms for proposing major new
+features, for collecting community input on an issue, and for
+documenting the design decisions that have gone into Jenkins. The JEP
+author is responsible for building consensus within the community and
+documenting dissenting opinions.
+
+Because the JEPs are maintained as text files in a versioned
+repository, their revision history is the historical record of the
+feature proposal footnoteref:[repo, https://github.com/jenkinsci/jep].
+
+
+== JEP Types
+
+There are three kinds of JEP:
+
+. A **Standards Track** JEP describes a new feature or implementation
+ for Jenkins. It may also describe an interoperability or
+ backwards-compatibility standard which will be supported for the feature in
+ current versions of Jenkins, moving forward.
+. An **Informational** JEP describes a Jenkins design issue, or
+ provides general guidelines or information to the Jenkins community,
+ but does not propose a new feature. Informational JEPs do not
+ necessarily represent a Jenkins community consensus or
+ recommendation, so users and implementers are free to ignore
+ Informational JEPs or follow their advice.
+. A **Process** JEP describes a process surrounding Jenkins, or
+ proposes a change to (or an event in) a process. Process JEPs are like
+ Standards Track JEPs but apply to areas other than the Jenkins codebase
+ itself. They may propose an implementation, but not for what would be
+ generally considered the Jenkins codebase; they often require community
+ consensus; unlike Informational JEPs, they are more than recommendations, and
+ users are typically not free to ignore them. Examples include procedures,
+ guidelines, changes to the decision-making process, and changes to the tools
+ or environment used in Jenkins development. Any meta-JEP (JEP focusing on the
+ improvement of the Jenkins Enhancement Proposal process) is also considered a
+ Process JEP.
+
```

Comment:
Suggest creating a `== JEP Terminology` here.  and move "Jenkins's BDFL" and "JEP Editors" under it.